core/io/config_file.cpp

```diff
@@ -164,6 +206,48 @@ Error ConfigFile::load(const String &p_path) {
 	if (!f)
 		return ERR_CANT_OPEN;
 
+	return _internal_load(p_path, f);
+}
+
+Error ConfigFile::load_encrypted(const String &p_path, const Vector<uint8_t> &p_key) {
+
+	Error err;
+	FileAccess *f = FileAccess::open(p_path, FileAccess::READ, &err);
+
+	if (err)
+		return err;
+
+	FileAccessEncrypted *fae = memnew(FileAccessEncrypted);
+	err = fae->open_and_parse(f, p_key, FileAccessEncrypted::MODE_READ);
```

Comment:
You should check that `fae != NULL` before dereferencing it, otherwise it could crash here. (Same below.)

Reply:
I just copied from https://github.com/godotengine/godot/blob/master/core/bind/core_bind.cpp#L1764
That code have no checking for NULL

Reply:
Right, nevermind. It's using `memnew` so it should be a valid pointer, contrarily to `FileAccess *f = FileAccess::open(p_path, FileAccess::READ, &err);` which can get NULL if it fails. So all good.